tests/test_perf_async_runs.py

```
@@ -0,0 +1,62 @@
+        # Simulate blocking I/O
+        time.sleep(0.5)
+        return []
+
+@pytest.mark.anyio
+async def test_list_runs_non_blocking():
+    """Verify that list_runs does not block the event loop."""
+
+    # We use a heartbeat coroutine to detect blocking
+    heartbeats = []
+
+    async def heartbeat():
```

Ruff (0.15.2)

[warning] Missing return type annotation for private function `heartbeat`

Add return type annotation: `None`

(ANN202)